### DISCUSSION TITLE
Import function for ZIA/ZPA

### DISCUSSION THREAD

**Discussion by 17yuyu-irdnz** [zscaler/terraform-provider-zia#49] [Q&A] [answered]
Hi Will,

Just wondering if the ZIA/ZPA provider can do import from existing setup?

Thanks

**Comment by willguibr** [comment, marked as answer]
Hi @17yuyu-irdnz The latest versions for both ZPA and ZIA providers now support the ability to import existing resources via name or numeric ID.
[ZPA - v2.0.5](https://github.com/willguibr/terraform-provider-zpa/releases/tag/v2.0.5)
[ZIA - v1.0.3](https://github.com/willguibr/terraform-provider-zia/releases/tag/v1.0.3)

Bear in mind that terraform import does not allow bulk import for resources though.
